src/core/thread/mle.cpp

```diff
@@ -64,6 +64,10 @@ namespace Mle {
 
 RegisterLogModule("Mle");
 
+const otMeshLocalPrefix Mle::sMeshLocalPrefixInit = {
+    {0xfd, 0xde, 0xad, 0x00, 0xbe, 0xef, 0x00, 0x00},
```

Comment:
Is fd00:: simpler value?

Reply:
I chose the current value to be consistent with the existing behavior.